### DISCUSSION TITLE
Mar 3rd week sprint

### DISCUSSION THREAD

**Discussion by zangsu** [zangsu/cs-test-service#6] [Sprint]
3월 셋째주 개인 과업 목표를 설정해서 올려주세요!

**Comment by zangsu** [comment]
# 백엔드 주간 목표
+ 사용자 정보 전송에 따른 세션 생성 로직 구현
+ 사용자 정보와 입력 정답을 저장할 수 있는 내부 DB 구현
+ 각 HTTP 통신에 해당하는 서블릿 생성
+ POSTMAN을 이용한 URI 검증

# 일간 목표
+ 세션 로직 구현 : ~ 15 (수)
+ 내부 DB 구현 : ~ 16 (목)
+ 서블릿 생성 &  URI 테스트 작성 : ~ 18 (토) 
+ URI 검증 완료 : ~ 19 (일)

**Comment by Moon-GD** [comment]
# 프론트엔드 주간 목표
+ 메인 화면 와이어 프레임 구성
+ 문제 화면 와이어 프레임 구성
+ 결과 화면 와이어 프레임 구성

# 일간 목표
+ 메인 화면 와이어 프레임 구성 : ~ 16 (목)
+ 문제 화면 와이어 프레임 구성 : ~ 18 (토)
+ 결과 화면 와이어 프레임 구성 : ~ 19 (일)
